local/network.go

```diff
@@ -171,6 +177,7 @@ func NewNetwork(
 	redirectStdout bool,
 	redirectStderr bool,
 	walletPrivateKey string,
+	genesisPath string,
```

Comment:
network config already has the genesisData (or should)

Reply:
k